sync_note.py

```diff
@@ -106,14 +94,59 @@ def sync_down(keep):
         keep.sync()
```

Comment:
@scdekov We need it because the library `gkeepapi` makes the request in such a way that we receive some kind of Timeout error and another kind of error is raised in the same time.